Google.Api.Gax.Grpc/Rest/RestChannel.cs

```diff
@@ -65,9 +65,9 @@ public RestChannel(RestServiceCollection serviceCollection, string endpoint, Cha
             var restMethod = _serviceCollection.GetRestMethod(method);
 
             var deadlineToken = CreateDeadlineCancellationToken(options, method.FullName);
-            var methodDisposalCts = new CancellationTokenSource();
+            var methodDisposalCts = CancellationTokenSource.CreateLinkedTokenSource(options.CancellationToken, deadlineToken);
```

Comment:
nit: methodDisposalCts => callCancellationTokenSource (just so that is the same as in the streaming call)

Reply:
Done.